client/my-sites/stats/stats-notices/all-notice-definitions.ts

```diff
@@ -45,12 +45,7 @@ const ALL_STATS_NOTICES: StatsNoticeType[] = [
 			hasPWYWPlanOnly,
 			shouldShowPaywallNotice,
 		}: StatsNoticeProps ) => {
-			// Show the notice only if the site is commercial.
-			if ( ! isCommercial ) {
-				return false;
-			}
-
-			if ( isVip ) {
+			if ( ! isCommercial || isVip || hasPaidStats ) {
```

Comment:
It seems `hasPaidStats` would cause L61 to be never reached again for PWYW plans 🤔

Reply:
Good catch! Fixed in [3b9aae5](https://github.com/Automattic/wp-calypso/pull/92884/commits/3b9aae59bad5d785aaba4e1b08ac15cbeae3d2ac).